packages/flutter/lib/src/widgets/bottom_navigation_bar_item.dart

```diff
@@ -31,11 +31,11 @@ class BottomNavigationBarItem {
       'This feature was deprecated after v1.19.0.'
     )
     this.title,
-    this.label,
+    @required this.label,
```

Comment:
The label is not required. Instead of it, you may also provide a (now deprecated) title.

Reply:
Without `title` or `label` field the bottomNabBarItem gives error, so why don't we just make `label` as required. It'll prevent them from getting this error.